tests/kafkatest/services/verifiable_producer.py

```diff
@@ -89,6 +89,9 @@ def num_not_acked(self):
 
     def stop_node(self, node):
         node.account.kill_process("VerifiableProducer", allow_fail=False)
+        if self.worker_threads is None:
```

Comment:
This is fine to leave like this since this is the current behavior of `BackgroundThreadService` in ducktape, but shall we file an upstream issue/patch to make it always have at least an empty list? This is annoying to have to have in here, and it's actually quite weird how `BackgroundThreadService` *starts* with an empty list, and only when you call `wait()` does it set it to `None`.